flokk_src/lib/_internal/components/fading_index_stack.dart

```diff
@@ -8,9 +7,9 @@ class FadingIndexedStack extends StatefulWidget {
   final Duration duration;
 
   const FadingIndexedStack({
-    Key key,
-    this.index,
-    this.children,
+    Key? key,
+    this.index = 0,
+    this.children = const <Widget>[],
```

Comment:
I think maybe both of these should be required rather than defaults? Is there anywhere we're not passing children or setting an index?